ossl/src/lib.rs

```diff
@@ -704,6 +710,17 @@ impl<'a> OsslParam<'a> {
         }))
     }
 
+    /// Creates an `OsslParam` from an OpenSSl provider pointer that contains
+    /// allocated data that can be safely zeroized before freeing the pointer
+    /// Use with care as it will overwrite any data pointed by the params
```

Comment:
```suggestion
    /// allocated data that can be safely zeroized before freeing the pointer.
    /// Use with care as it will overwrite any data pointed by the params.
```